lib/sort_by_length.rb

```diff
@@ -1,7 +1,19 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2)
+# Space complexity: O(1)
 def sort_by_length(my_sentence)
```

Comment:
👍 Nice insertion sort.  However since you're doing `.split` this is O(n) space complexity.